nautobot/core/views/mixins.py

```diff
@@ -747,7 +747,10 @@ def _process_create_or_update_form(self, form):
             msg = f'{"Created" if object_created else "Modified"} {queryset.model._meta.verbose_name}'
             self.logger.info(f"{msg} {obj} (PK: {obj.pk})")
             if hasattr(obj, "get_absolute_url"):
-                msg = format_html('{} <a href="{}">{}</a>', msg, obj.get_absolute_url(), obj)
+                try:
+                    msg = format_html('{} <a href="{}">{}</a>', msg, obj.get_absolute_url(), obj)
+                except AttributeError:
+                    msg = format_html("{} {}", msg, obj)
             else:
                 msg = format_html("{} {}", msg, obj)
```

Comment:
Wondering if we even need the `hasattr` check now, or if we could just do the try/except in all cases?